bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/activities/AbstractActivityManager.java

```diff
@@ -22,6 +22,8 @@
 public abstract class AbstractActivityManager implements IActivityManager {
 	private ListenerList<IActivityManagerListener> activityManagerListeners;
 
+	private boolean isDisposed = false;
```

Comment:
IMO `disposed` would be better, otherwise it looks like a method.

Reply:
Seeing this code, can we please don't init it to default value?
`private boolean disposed;` should be enough

Reply:
agree with both comments, fixed with ada4186